src/embedded.js

```diff
@@ -645,9 +650,15 @@
             XWM.receive(function _parentWindowCallback(evt){
                 var source = evt.source || 'hsEmbeddedFrame';
 
+                // DOM elements cannot be stringified; regular elements resolve to {}, React elements throw errors
+                const safeParams = Object.keys(params).reduce((safe, key) => {
```

Comment:
@GreyeSmyth Our build process doesn't pipe this through babel, so we're not using ES6 at the moment. Can you remove the fat arrows and `const`s?

Reply:
Done, sorry about that.